tests/test_schema_validator.py

```
@@ -0,0 +1,94 @@
+import pytest
+from unittest.mock import MagicMock, patch
+from utils import schema_validator
+
+import pytest
```

Ruff (0.11.9)

Redefinition of unused `pytest` from line 1

Remove definition: `pytest`

(F811)